src/accordion/Accordion.test.html

```diff
@@ -0,0 +1,29 @@
+`
+<div id="accordion">
```

Comment:
is there a reason why we don't just use an existing part of our site to test this? just copy paste an html snippet out of somewhere where we use accordian?